swap_meet/item.py

```diff
@@ -1,2 +1,27 @@
+import uuid
+
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id = None, condition = 0):
+        if id is None:
+            self.id = uuid.uuid4().int
+        else:
+            self.id = id
+        self.condition = condition        
+
+    
+    def get_category(self):
+        return "Item"
+    
+    def __str__(self):
+        return f"An object of type Item with id {self.id}."
```

Comment:
@sourika and @SophieQA, recall that in Wave 05 instructions, you  want all your children classes (`Decor`, `Electronic`, and `Clothing`)  of `Item` to have a stringify method that returns this pattern of string:  `An object of type <TYPE> with id <id value>.  <String specific to Child Class>`

THe goal here is leverage inheritance write the first part of this string with the parent class's `__str__` method while also using the child's `__str__` method to get the second half. Right now in your child classes you have redundancies because you aren't following this pattern.  Please refactor your classes leveraging inheritance to complete this functionality. Once completed, message me on Slack and I can change the grade to a green.